pkgs/tools/security/asc-key-to-qr-code-gif/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, imagemagick, libqrencode
+, testQR ? false, zbar ? null
+}:
+
+assert testQR -> zbar != false;
+
+stdenv.mkDerivation rec {
+  name = "asc-key-to-qr-code-gif-${version}";
+  version = "e5d517b9fe230b9814458b84a9559358f9a492aa"; # no proper versions
```

Comment:
In the case of a project like this with no releases, we typically use the date the commit was made. For example, if the revision was commited on Jun 11, 2017, we'd use 20170611 as the version. This would let us have increasing version numbers when we make updates to the package. 

By the way, it looks like there were a lot more commits today.